automated_testing/testcases/new-config-is-gossiped/run_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+hab pkg install core/jq-static -b -f
+
+hab svc load core/redis --remote-sup=alpha.habitat.dev
+sleep 15
+hab svc load christophermaier/test-probe --bind=thing_with_a_port:redis.default --remote-sup=beta.habitat.dev
```

Comment:
@christophermaier How do you feel about creating a `habitat-testing` or similarly named origin (or maybe just use the `habitat` origin?) and pulling this package into it, along with moving the source to the `habitat-sh` org on github?

Reply:
Yes, I want to do this... I think I'll get this after we initially merge, so I can unblock @mwrock